pkg/kubelet/cpumanager/cpu_manager.go

```diff
@@ -88,6 +94,20 @@ func NewManager(policyName PolicyName, cr internalapi.RuntimeService, kletGetter
 	}, nil
 }
 
+func getReserverdCores(nodeStatus v1.NodeStatus) int64 {
+	// Kube/System Reserved = Capacity - Allocatable
+	capacity := nodeStatus.Capacity[v1.ResourceCPU]
+	reserved := capacity.Copy()
+	reserved.Sub(nodeStatus.Allocatable[v1.ResourceCPU])
+	glog.Infof("[cpumanager] %v cores reserved for kube/system and shared pool", reserved.Value())
+	if reserved.IsZero() {
+		glog.Infof("[cpumanager] defaulting to 1 core")
```

Comment:
That sounds great Seth, there's a little blurb in the proposal about adding taints in the static manager scenarios (shared pool becomes empty, shared pool becomes nonempty) that could use some expansion.